requirements/test.txt

```diff
@@ -4,3 +4,4 @@ pytest
 pytest-django
 pytest-cov
 coverage
+tox
```

Comment:
I think it belongs in dev.txt as a dev utility -- and in .travis.yml or any other CI config, it's installed individually.. but test.txt is for the CI's auto-configured test environments which are managed by tox.